java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -337,4 +340,32 @@ public void waitTest() {
         assertEquals(expectedResult[0], results[0]);
         assertTrue((Long) expectedResult[1] <= (Long) results[1]);
     }
+
+    @Test
+    @SneakyThrows
+    public void test_transaction_function_dump_restore() {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0"));
+        String libName = "mylib";
+        String funcName = "myfun";
+        String code = generateLuaLibCode(libName, Map.of(funcName, "return args[1]"), true);
+
+        // Setup
+        clusterClient.functionLoad(code, true).get();
+        clusterClient.functionList(true).get();
+
+        // Verify functionDump
+        ClusterTransaction transaction1 = new ClusterTransaction();
+        transaction1.withBinarySafeOutput();
+        transaction1.functionDump();
+        Object[] result = clusterClient.exec(transaction1).get();
+        GlideString payload = (GlideString) (result[0]);
+
+        // Verify functionRestore
+        ClusterTransaction transaction2 = new ClusterTransaction();
+        transaction2.withBinarySafeOutput();
```

Comment:
is this necessary?  Restore returns "OK".

Reply:
This one is safe to remove.